java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/protocol/match/MatchField.java

```diff
@@ -191,6 +192,15 @@ private MatchField(final String name, final MatchFields id, Prerequisite<?>... p
 
     public final static MatchField<OFBooleanValue> PBB_UCA =
             new MatchField<OFBooleanValue>("pbb_uca", MatchFields.PBB_UCA);
+    
+    public final static MatchField<U16> TCP_FLAGS =
```

Comment:
@abakagamze For these, there should be a prerequisite MatchField defined for IpProtocol.TCP (see TUNNEL_IPV4_SRC below for example on how to define a prerequisite)